app/graphql/queries.js

```diff
@@ -215,6 +215,7 @@ export const GET_ALL_STATUS = gql`
           equipmentCountPerOui
           clientCountPerOui
         }
+        alarmsCount
```

Comment:
remove this and create a new gql query. Doing this will cause you to load a lot of unused data when getting just the alarmsCount, and will cause alarmsCount to load on every other call of GET_ALL_STATUS